core/src/main/java/org/stellar/anchor/model/Sep38Quote.java

```diff
@@ -0,0 +1,61 @@
+package org.stellar.anchor.model;
+
+import java.time.LocalDateTime;
+
+public interface Sep38Quote {
+  String getId();
+
+  void setId(String id);
+
+  LocalDateTime getExpiresAt();
+
+  void setExpiresAt(LocalDateTime expiresAt);
+
+  String getPrice();
+
+  void setPrice(String price);
+
+  String getSellAsset();
+
+  void setSellAsset(String sellAsset);
+
+  String getSellAmount();
+
+  void setSellAmount(String sellAmount);
+
+  String getSellDeliveryMethod();
+
+  void setSellDeliveryMethod(String sellDeliveryMethod);
+
+  String getBuyAsset();
+
+  void setBuyAsset(String buyAsset);
+
+  String getBuyAmount();
+
+  void setBuyAmount(String buyAmount);
+
+  String getBuyDeliveryMethod();
+
+  void setBuyDeliveryMethod(String buyDeliveryMethod);
+
+  LocalDateTime getCreatedAt();
+
+  void setCreatedAt(LocalDateTime createdAt);
+
+  String getCreatorAccountId();
+
+  void setCreatorAccountId(String creatorAccountId);
+
+  String getCreatorMemo();
+
+  void setCreatorMemo(String creatorMemo);
+
+  String getCreatorMemoType();
+
+  void setCreatorMemoType(String creatorMemoType);
```

Comment:
❓ I know in the Platform API we use the term "creator", but do we use this pattern for the `SEP31Transaction` model? I'd rather have a consistent naming convention within the anchor SDK and only use the "creator" term in the Platform API.

Reply:
I had not started the `Sep31Transaction` model yet. But I would probably follow the SEP document. I will keep this in mind when I encounter this.